language/move-lang/src/lib.rs

```diff
@@ -40,6 +40,7 @@ use tempfile::NamedTempFile;
 
 pub const MOVE_EXTENSION: &str = "move";
 pub const MOVE_COMPILED_EXTENSION: &str = "mv";
+pub const MOVE_COMPILED_INTERFACES_DIR: &str = "mv_interfaces";
```

Comment:
Alternative name ideas: `.mvi` or `.movei` (since it's basically a source file).

Reply:
movei isn't bad, but right now the compiler doesn't actually treat them differently than a move file, so for now I don't think its worthwhile to split

Reply:
Oh, I somehow missed that this was a dir! Totally agree with `.move` to keep it simple for the compiler.